src/OpenLoco/src/Windows/CompanyList.cpp

```diff
@@ -655,14 +660,14 @@ namespace OpenLoco::Ui::Windows::CompanyList
 
     namespace CompanyPerformance
     {
-        static constexpr Ui::Size kWindowSize = { 635, 322 };
+        static constexpr Ui::Size32 kWindowSize = { 635, 322 };
 
         const uint64_t enabledWidgets = Common::enabledWidgets;
 
-        static constexpr Widget widgets[] = {
-            commonWidgets(635, 322, StringIds::title_company_performance),
-            widgetEnd(),
-        };
+        static constexpr auto widgets = makeWidgets(
+            Common::makeCommonWidgets(635, 322, StringIds::title_company_performance)
+
```

Comment:
There's a few stray newlines in this file in places where `widgetEnd()` used to be. Also in AboutMusic.cpp I think.

Reply:
The thing with clang-format is that when there is no new-line like that it won't really look like a nice table, it will become
```cpp
static constexpr auto widgets = makeWidgets(
            Common::makeCommonWidgets(635, 322, StringIds::title_company_performance));
```
But I don't really mind removing the empty lines, I just thought its a bit easier to work with and keeping diffs smaller when one modifies them.

Reply:
Ahh, I thought it was an oversight, tbh. In that case, though, let's just leave it.